src/Stryker.Core/Stryker.Core/Reporters/Json/JsonReport.cs

```diff
@@ -12,25 +14,24 @@ public class JsonReport
         public IDictionary<string, int> Thresholds { get; init; } = new Dictionary<string, int>();
         public string ProjectRoot { get; init; }
         public IDictionary<string, SourceFile> Files { get; init; } = new Dictionary<string, SourceFile>();
+        public IDictionary<string, JsonTestFile> TestFiles { get; set; } = null;
```

Comment:
Sorry, I'm a null-paranoid, why not assign this an empty collection as well? :D

Reply:
If there is no test information for this run, the result should be that the json does not contain a property for tests, instead of 'there are no tests'. That is imo semantically different.

Reply:
:thumbsup: